assets/license.rtf

```diff
@@ -1,390 +1,389 @@
-{\rtf1\adeflang1025\ansi\ansicpg1252\uc1\adeff0\deff0\stshfdbch0\stshfloch31506\stshfhich31506\stshfbi31506\deflang1033\deflangfe1033\themelang1033\themelangfe0\themelangcs0{\fonttbl{\f0\fbidi \froman\fcharset0\fprq2{\*\panose 02020603050405020304}Times New Roman;}{\f2\fbidi \fmodern\fcharset0\fprq1{\*\panose 02070309020205020404}Courier New;}

```

Comment:
Please don't change this file

Reply:
If something needs changes, please open an issue, with images of what needs to be changed

Reply:
Issue created: #8691

Reply:
Fixed in #8646